packages/react/etc/react.api.md

```diff
@@ -6136,6 +6140,8 @@ export type IGroupFooterStyleProps = Required<Pick<IGroupFooterProps, 'theme'>>
 export interface IGroupFooterStyles {
     // (undocumented)
     root: IStyle;
+    // (undocumented)
+    title: IStyle;
 }
 
 // @public (undocumented)
```

Comment:
I believe that there should be additional typescript changes.

IDetailsItemProps has groupNestingDepth defined as "Nesting depth of a grouping" yet

IDetailsHeaderProps and IDetailsFooterProps both extend IDetailsItemProps and the DetailsList supplies groupNestingDepth as the "total nesting depth" when calling the render functions.  ( DetailsHeader then aligns ) 

IDetailsGroupDividerProps also extends and again DetailsList supplies "total nesting depth" for the custom renderers of the group header and footer. ( The groupNestingDepth is not used by the GroupHeader and GroupFooter )


----
IDetailsRowProps also extends - it is the "level of the group" which is used for aria and as such agrees with the comment. ( The GroupedListSection passes this to the GroupedList onRenderCell prop function.  DetailsList will call onRenderRow with IDetailsRowProps groupNestingDepth set to this value )